### ISSUE TITLE
[GUI] Partial Context menu

### ISSUE THREAD

**Issue by rprimus** [joaotavora/sly#181]
Mon Jul 16 18:07:25 BST 2018

<details>
<summary>System Info:</summary>
<hr>

### System Info:

- **OS**:
  - ProductName:	Mac OS X
  - ProductVersion:	10.13.6
  - BuildVersion:	17G65
- **Emacs**: GNU Emacs 26.1 (build 1, x86_64-apple-darwin17.7.0, Carbon Version 158 AppKit 1561.4) of 2018-06-14
- **Lisp**: SBCL 1.4.9
- **sly**: sly-20180715.603
<hr>
</details>
<br>

Am going over `3. A SLY tour for SLIME users` using a non-terminal (GUI) Emacs.

Initially,  when right-clicking on a `button`,  the complete context menu (including the equivalent keybindings) is presented.

If the user then clicks anywhere else in the frame and then returns to the button,  the context menu presented (via right-click) does not contain the equivalent keybindings.

Is this by design?

![sly-menu](https://user-images.githubusercontent.com/278414/42772700-1fc29948-8923-11e8-8bd8-dc9691775170.gif)

**Comment by joaotavora** [comment]
> Is this by design?

No.  But it doesn't happen in my gtk build, so seems to be osx specific. A bug report should be made to Emacs using `M-x report-emacs-bug`, but perhaps containing a much shorter reproduction recipe.  Something that pops a menu with a couple of commands bound to keys.

If you want to report it for SLY, you *can* mention it there, provided you explain in the recipe how to install it and see the problem.  Analysis usually takes longer in those cases.

I'll try to come up with a simple elisp recipe if you can't.

**Comment by rprimus** [comment]
Tue Jul 17 00:18:44 BST 2018

Will have a look around.

**Comment by rprimus** [comment]
Tue Jul 17 06:18:42 BST 2018

From `NEWS.md`:

> Following a suggestion by Attile Lendvai to SLIME, mouse-1 is now activates SLY's "part buttons".

What does the above mean/refer to?

**Comment by joaotavora** [comment]
> Am experiencing the same:

Funny, I'm not.  

Can you try this much simpler exercise?

```
emacs -Q
C-mouse-3 ;; hold down control and click the right mouse button
```

Do it a couple of times and tell me if the tooltips disappear the second time.

**Comment by rprimus** [comment]
Wed Jul 18 05:50:29 BST 2018

[Using macOS]:

Nope.   In the `*scratch*` buffer, `C-mouse-3` brings up the `Lisp Interaction Mode` tooltip (everytime).

![emacs-q-c-mouse-3](https://user-images.githubusercontent.com/278414/42860789-174d415e-8a50-11e8-8073-def8b89fbe8c.gif)
